src/05-led-roulette/debug-it.md

```diff
@@ -10,9 +10,9 @@ The starter project I've provided to you has some extra code that runs *before*
 At this time, we are not interested in that "pre-main" part so let's skip right to the beginning of
 the `main` function. We'll do that using a breakpoint. Issue `break main` at the `(gdb)` prompt:
 
-> **Note** for these gdb commands I generally won't provide a copyable code block
+> **NOTE** For these GDB commands I generally won't provide a copyable code block
```

Comment:
I forgot this change in my previous commit. Now it is updated accordingly. Sorry for that!